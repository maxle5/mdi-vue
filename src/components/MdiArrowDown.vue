
<template>
  <svg :width="size" :height="size" viewBox="0 0 24 24">
      <path d="M11,4H13V16L18.5,10.5L19.92,11.92L12,19.84L4.08,11.92L5.5,10.5L11,16V4Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiArrowDown",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
  })
</script>
