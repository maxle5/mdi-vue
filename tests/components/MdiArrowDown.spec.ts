
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiArrowDown from "../../src/components/MdiArrowDown.vue";

test("MdiArrowDown snapshot", () => {
  const wrapper = mount(MdiArrowDown, {});
  expect(wrapper.html()).toMatchSnapshot();
});
