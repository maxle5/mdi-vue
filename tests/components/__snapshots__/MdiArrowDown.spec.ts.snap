// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiArrowDown snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M11,4H13V16L18.5,10.5L19.92,11.92L12,19.84L4.08,11.92L5.5,10.5L11,16V4Z" style="fill: currentColor;"></path>
</svg>"
`;
